### DISCUSSION TITLE
[Script request] Netboot.XYZ

### DISCUSSION THREAD

**Discussion by TazztheMonster** [community-scripts/ProxmoxVE#819] [Request script]
### Application Name

Netboot.XYZ

### Website

https://github.com/netbootxyz/netboot.xyz

### Description

Netboot.XYZ is a simple and powerful PXE boot tool that provides access to various OS installers and utilities over the network. Adding it to Proxmox VE Helper Scripts would streamline network boot setup, making OS deployments and troubleshooting easier.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
https://github.com/community-scripts/ProxmoxVED/issues/1601
